code/modules/client/client_defines.dm

```diff
@@ -111,6 +111,9 @@
 	/// Is the client watchlisted
 	var/watchlisted = FALSE
 
+	/// Makes sure it can return to a blank status bar text when done
+	var/status_bar_prev_text = ""
+
```

Comment:
This var appears to be unused.

Reply:
Ah, thank you! It became unsued when I ported https://github.com/tgstation/tgstation/pull/63246
but I forgot to remove this biy